stock_inventory_quantity_history/views/stock_move_line_views.xml

```diff
@@ -0,0 +1,45 @@
+<odoo>
+    <record id="stock_move_line_form" model="ir.ui.view">
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_move_line_form" />
+        <field name="arch" type="xml">
+            <field name="lot_id" position="before">
+                            <label
+                    for="inventory_theoretical_qty"
+                    string="Theoretical Quantity"
+                />
+                            <div class="o_row">
+                                <field name="inventory_theoretical_qty" />
+                                <field
+                        name="product_uom_id"
+                        options="{'no_create': True}"
+                        string="Unit of Measure"
+                        groups="uom.group_uom"
+                    />
+                            </div>
+                            <label for="inventory_real_qty" string="Real Quantity" />
+                            <div class="o_row">
+                                <field name="inventory_real_qty" />
+                                <field
+                        name="product_uom_id"
+                        options="{'no_create': True}"
+                        string="Unit of Measure"
+                        groups="uom.group_uom"
+                    />
+                            </div>
```

Comment:
Indentations look distorted here. Can you please manually fix them?

Reply:
Done